src/main/java/gregtech/common/terminal/app/teleport/TeleportApp.java

```diff
@@ -0,0 +1,87 @@
+package gregtech.common.terminal.app.teleport;
+
+import gregtech.api.gui.resources.ColorRectTexture;
+import gregtech.api.gui.widgets.ClickButtonWidget;
+import gregtech.api.gui.widgets.ImageWidget;
+import gregtech.api.gui.widgets.TextFieldWidget2;
+import gregtech.api.terminal.app.AbstractApplication;
+import gregtech.api.terminal.os.SystemCall;
+import gregtech.api.terminal.os.TerminalTheme;
+import gregtech.api.util.TeleportHandler;
+import gregtech.common.entities.PortalEntity;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.world.chunk.Chunk;
+
+public class TeleportApp extends AbstractApplication {
+
+    private int CoordinateX = 0;
+    private int CoordinateY = 1;
+    private int CoordinateZ = 0;
+
+    private int Dimension = 0;
```

Comment:
Conventions

Reply:
wdym?

Reply:
We use lower case var names is what rong meant i think.

Reply:
Alright, will change that